Library/Homebrew/cask/lib/hbc/utils.rb

```diff
@@ -145,8 +145,9 @@ def self.path_occupied?(path)
 
     def self.error_message_with_suggestions
       <<-EOS.undent
-        Most likely, this means you have an outdated version of Homebrew-Cask. Please run:
-          #{UPDATE_CMD}
+
```

Comment:
Yes, that sounds like more of an issue with the call site than with the message itself.